ui/components/app/snaps/snap-delineator/index.scss

```diff
@@ -8,8 +8,13 @@
     &__text {
       white-space: nowrap;
       text-overflow: ellipsis;
+      width: calc(100% - 16px);
```

Comment:
Curious why this is suddenly necessary?

Reply:
Had to do all of this to fix the header.

Reply:
We never tested the delineator title that would overflow